src/styles/card.css

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  font-size: 12px;
+  color: var(--main-text-color);
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  color: var(--second-text-color);
+  font-weight: 400;
```

Comment:
But it's a regular value, isn't it? Set it to `html` tag and override if some element will need it

Reply:
Actually it's a default value, so remove it